.travis/test.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/bash
 
 error=0
-for version in '5.6' '7.1' '7.2' '7.3'; do
+for version in '5.6' '7.1' '7.2' '7.3' '7.4'; do
```

Comment:
I think we're going to have to remove 5.6 and 7.1 from here otherwise the tests will fail.